apps/web/pages/settings/my-account/profile.tsx

```diff
@@ -363,6 +306,73 @@ const ProfileView = () => {
   );
 };
 
+const ProfileForm = ({
```

Comment:
ProfileForm only cares for ProfileForm. IMO it adds extra legibility. Also a Form should not fetch data. That's what containers are for.